aws/customizations/workspace/solution_accelerators/system_tables_audit_log/system_table_audit_log.md

```diff
@@ -0,0 +1,27 @@
+#### Audit Log Alerting: 
+Audit Log Alerting, based on this [blog post](https://www.databricks.com/blog/improve-lakehouse-security-monitoring-using-system-tables-databricks-unity-catalog), creates 40+ SQL alerts to monitor incidents using a Zero Trust Architecture (ZTA) model.
+
+- **NOTE:** Enabling this feature will create a cluster, a job, and queries within your environment.
+
+### How to add this resource to SRA:
+
+1. Copy the `system_tables_audit_log` folder into `modules/sra/databricks_workspace/` 
+2. Add the following code block into `modules/sra/databricks_workspace.tf`
+```
+module "system_tables_audit_log" {
+  source = "./databricks_workspace/system_tables_audit_log/"
+  providers = {
+    databricks = databricks.created_workspace
+  }
+
+  alert_emails = [var.user_workspace_admin]
+
+  depends_on = [
+    module.databricks_mws_workspace
```

Comment:
I encountered a race condition with `module.uc_assignment` causing this module to fail because my user ID wasn't assigned to the workspace yet.  Thoughts on adding `module.uc_assignment` to _depends_on_?